src/intl/en/page-what-is-ethereum.json

```diff
@@ -3,52 +3,49 @@
-  "page-what-is-ethereum-what-is-crypto-desc-2": "It all started in 2008 with Bitcoin. You could use it to send funds to anyone anywhere globally. What made crypto different from normal bank transfers or other financial services like Paypal or Alipay is that there was no middleman for the first time.",
-  "page-what-is-ethereum-what-is-crypto-desc-3": "Wait, what is a middleman?",
-  "page-what-is-ethereum-what-is-crypto-desc-4": "A middleman is a central authority like a bank or government that intervenes in a transaction between the sender and recipient. They have the power to surveil, censor or revert transactions and they can share the sensitive data they collect about you with third parties. They also often dictate which financial services you have access to.",
-  "page-what-is-ethereum-what-is-crypto-desc-5": "Things are different with crypto. <b>Transactions directly connect sender and recipient</b> without having to deal with any central authority. Nobody else will have access to your funds and <b>nobody can tell you what services you can use</b>. This is possible because of the blockchain technology upon which cryptocurrencies operate.",
+  "page-what-is-ethereum-summary-desc-1": "Ethereum is a network of computers all over the world that follow a set of rules called the Ethereum protocol. Ethereum network acts as the foundation for communities, applications, organizations and digital assets that anyone can build and use.",
+  "page-what-is-ethereum-summary-desc-2": "You can create an Ethereum account from anywhere, at any time, and explore a world of apps or build your own. The core innovation is that you can do all this without trusting a central authority that could change the rules or restrict your access.",
+  "page-what-is-ethereum-summary-desc-3": "Keep reading to learn more…",
   "page-what-is-ethereum-btc-eth-diff-title": "What is the difference between Ethereum and Bitcoin?",
   "page-what-is-ethereum-btc-eth-diff-1": "Launched in 2015, Ethereum builds on Bitcoin's innovation, with some big differences.",
   "page-what-is-ethereum-btc-eth-diff-2": "Both let you use digital money without payment providers or banks. But <b>Ethereum is programmable</b>, so you can also build and deploy decentralized applications on its network.",
   "page-what-is-ethereum-btc-eth-diff-3": "Bitcoin enables us to send basic messages to one another about what we think is valuable. Establishing value without authority is already powerful. Ethereum extends this: rather than just messages, you can write any general program, or contract. There is no limit to the kind of contracts which can be created and agreed upon, hence great innovation happens on the Ethereum network.",
   "page-what-is-ethereum-btc-eth-diff-4": "While Bitcoin is only a payment network, Ethereum is more like a marketplace of financial services, games, social networks and other apps that respect your privacy and cannot censor you.",
```

Comment:
yea agree this should change. Especially since the subjects of the sentence are in the application layer.

```suggestion
  "page-what-is-ethereum-btc-eth-diff-4": "While Bitcoin is only a payment network, Ethereum is more like a marketplace of financial services, games, social networks and other apps.",
```